packages/pwa/app/commerce-api/utils.test.js

```diff
@@ -29,6 +34,87 @@ describe('isTokenValid', () => {
         expect(isTokenValid(createJwt(59))).toBe(false)
     })
 
+    test('convertSnakeCaseToSentenceCase returns correct formatted string', () => {
+        const snakeCaseString = 'test_snake_case_string'
+        const expectedSentenceCaseString = 'test snake case string'
+
+        expect(convertSnakeCaseToSentenceCase(snakeCaseString) === expectedSentenceCaseString).toBe(
+            true
+        )
+    })
```

Comment:
do you mind moving this test case out from `isTokenValid` test group, and create a new group for this function? This makes it easier to locate the tests.

like:

```
describe('convertSnakeCaseToSentenceCase', () => {
  test('convertSnakeCaseToSentenceCase returns correct formatted string', () => {
        const snakeCaseString = 'test_snake_case_string'
        const expectedSentenceCaseString = 'test snake case string'

        expect(convertSnakeCaseToSentenceCase(snakeCaseString) === expectedSentenceCaseString).toBe(
            true
        )
    })
})
```

Reply:
yup no problem